substrate/frame/support/src/traits/tokens/fungible/conformance_tests/regular/unbalanced.rs

```diff
@@ -146,7 +146,7 @@ where
 			Preservation::Preserve,
 			Fortitude::Polite,
 		),
-		Err(TokenError::FundsUnavailable.into()),
+		Err(TokenError::BelowMinimum.into()),
```

Comment:
Can we also add a conformance test for FundsUnavaliable?

Reply:
Yes I will add one